server_addon/maya/client/ayon_maya/plugins/load/load_reference.py

```diff
@@ -124,6 +124,7 @@ class ReferenceLoader(plugin.ReferenceLoader):
         "skeletalMesh",
         "mvLook",
         "matchmove",
+        "OxCache",
```

Comment:
Should still turn lowercase?
```suggestion
        "oxcache",
```